global-api/routes/city_locode_endpoint.py

```diff
@@ -0,0 +1,135 @@
+from fastapi import FastAPI
+from sqlalchemy import create_engine, text 
+from sqlalchemy.orm import sessionmaker
+
+app = FastAPI()
```

Comment:
I think we should have one shared FastAPI app for all routes? I haven't checked the docs though, if this is recommended then go for it.